docker/yb-voyager-docker

```diff
@@ -95,6 +95,95 @@ do
 		fi
 	fi	
 
+	# If the flag is --source-ssl-cert then create a volume for the cert directory and map it to the container
+	if [[ ${argv[${i}]} == "--source-ssl-cert" ]]
+	then
+		j=$(( $i + 1))
+		source_ssl_cert=${argv[${j}]}
+		source_ssl_cert_dir=$(dirname "$source_ssl_cert")
+		source_ssl_cert_dir=$(realpath "$source_ssl_cert_dir")
+		source_ssl_cert_file=$(basename "$source_ssl_cert")
+		if [[ $OSTYPE == 'darwin'* ]]
+		then
+			docker volume rm voyager-source-ssl-cert-dir
+			docker volume create --driver local --opt type=none --opt device="${source_ssl_cert_dir}" --opt o=bind voyager-source-ssl-cert-dir
+			source_ssl_cert_vol="-v voyager-source-ssl-cert-dir:/source-ssl-cert-dir"
+		else
+			source_ssl_cert_vol="-v ${source_ssl_cert_dir}:/source-ssl-cert-dir"
+		fi
+		argv[$j]='/source-ssl-cert-dir/'$source_ssl_cert_file
```

Comment:
See if you can factor out this in a separate function named `map_path_inside_container(path, path_in_container)`